lib/rom/csv/commands/create.rb

```diff
@@ -0,0 +1,31 @@
+require 'rom/commands'
+require 'rom/commands/create'
+
+module ROM
+  module CSV
+    module Commands
+      class Create < ROM::Commands::Create
+        def execute(tuples)
+          insert_tuples = Array([tuples]).flatten.map do |tuple|
```

Comment:
I will leave [tuples].flatten.map

Array(tuples) doesn't work the same like [tuples] and flatten.map doesn't work the same like flat_map